template/build/webpack.prod.conf.js

```diff
@@ -1,4 +1,5 @@
 var path = require('path')
+var glob = require('glob-all')
```

Comment:
Not sure how to do better, seems like i pushed wrong commit for pull request. 
Can i merge all these commits to one or need close this pull request and create a new one?